azkaban-common/src/main/java/azkaban/metrics/MetricsManager.java

```diff
@@ -40,28 +41,41 @@
 @Singleton
 public class MetricsManager {
 
-  private static final Logger logger = Logger.getLogger(MetricsManager.class);
+  private static final Logger log = LoggerFactory.getLogger(MetricsManager.class);
```

Comment:
Yes. That's what @HappyRay is pushing for as well. And I think it has some advantages since we already have it in our dependency.